.github/workflows/ci.yml

```diff
@@ -32,5 +32,3 @@ jobs:
             --doctest-modules \
             --cov fits_schema --cov-report=xml \
             tests fits_schema
-
-      - uses: codecov/codecov-action@v1
```

Comment:
Do you want to add codecov back to this orga? If yes, you need to install the app (I don't have permission) and , add token to the CI, and update the action version to `@v5`

Reply:
No, I will in the next PR modernize all of this to the latest python_project_template.